src/manager/app/services/app_data.py

```diff
@@ -0,0 +1,29 @@
+import logging
+import os
+import shutil
+
+from app.models.volume import STATE_HOST_DIR
+from app.utils import check_app_name
+
+logger = logging.getLogger("uvicorn")
+# The default data directory (in container) for RAGapp data
+DEFAULT_RAGAPP_DATA_DIR = os.getenv("RAGAPP_DATA_DIR", "data/ragapps")
+
+
+class AppDataService:
+    _data_mount_path: str = "/app/data"
+
+    @staticmethod
+    def remove_app_data(
+        app_name: str,
+    ):
+        app_name = check_app_name(app_name)
+        if STATE_HOST_DIR is not None:
```

Comment:
STATE_HOST_DIR is not used below

Reply:
I think i added this just to safely remove if only manager has mounted with the data - but because we're now always mount the state so it's redundant. It's already removed. Thank you!